CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.1.1 - 2022-03-07
+
+### Added
+
+- Added support for Django config detection for different versions (PR #187)
+
+### Changed
+
+- Bumped `hashed_key` field's `max_length` from 100 to 150 to address length issue with `argon2-cffi` (PR #193)
+- Code clean-up for Django 3.2 (PR #190)
+- Bumped dependency versions and added official support for Django 3.2 and Python 3.9 and 3.10 (PR #189)
+- Updated contribution documentation (PR #183)
```

Comment:
I believe some of these non-user-impacting changes can be dropped, and this can boil down to:

```suggestion
- Add official support for Django 3.2 and Python 3.9 and 3.10 (PR #189)
```

Then we can move this to `Added`, what do you think?